### DISCUSSION TITLE
Feature Request: Four Finger Click

### DISCUSSION THREAD

**Discussion by fern185** [rxhanson/Middle-Localization#75] [Ideas]
Thanks for making Middletouch, it's really handy. 

I'm avoiding Three Finger Click because I already use Three Finger Drag, so I'm using Four Finger Tap. But I don't use Tap To Click (i.e., for normal left or right clicks), so Four Finger Tap feels weird to me. I'd like an option in Middletouch to use Four Finger Click. Is this possible?

**Comment by rxhanson** [comment]
Thanks for the feedback! This one's on my backlog, but it unfortunately it will be some time before I can get a chance to add it in. MiddleTouch is intentionally slim and came out of Multitouch, which has this gesture. If you have purchased MiddleTouch and are inclined to purchase Multitouch, send me an email and I'll provide a discount.

**Reply by fern185** [reply]
Hi Ryan, thanks for responding so promptly. I really appreciate the offer of a discount on Multitouch, that's very nice of you. I did have a look at Multitouch, but it looks like more app than I need; I chose Middletouch in part because of its slim nature. So, I think I'd rather be patient and wait for the feature to eventually come to Middletouch. Thanks again!
